### ISSUE TITLE
[BUG] Prayer of Mending crits are applying Divine Aegis in holy and shadow spec

### ISSUE THREAD

**Issue by BmblBii** [ClassicWoWCommunity/cata-classic-bugs#1483]
#### 1.1. Description
crits with prayer of mending are applying absorb shield in all specs even with no points in the disc talent. This is obviously very OP for shadow and holy, especially in PVP. Prayer of mending is also not affected by healing reduction or absorb effects such as chogal's absolute corruption

  #### 1.2. How to Reproduce

  #### 1.3  Source Material
![Screenshot 2024-08-11 212535](https://github.com/user-attachments/assets/2d45e7a8-c4d2-4a6e-bc86-c2eca5fe6d64)

### 2. Expected Behavior
 divine aegis should only proc for disc and only with the talent

  #### 2.2 Source Material

https://wowpedia.fandom.com/wiki/Prayer_of_Mending

**Comment by fkelava** [comment]
I can't reproduce this.

<https://classic.warcraftlogs.com/reports/pcbPFnRrVx1m9J37#type=healing&fight=1&view=events> in disc spec,
<https://classic.warcraftlogs.com/reports/pcbPFnRrVx1m9J37#type=healing&fight=3&view=events> in shadow spec

but I see a different bug- PoM crits are applying Aegis not only to the recipient of the critical heal, but also the casting priest, which seems incorrect - e.g. Flash Heal applying Aegis would only apply it to the recipient, so how is PoM different?

![image](https://github.com/user-attachments/assets/65cf9c0d-f5aa-4747-af27-5c7f3894e4cb)

**Comment by BmblBii** [comment]
dont know what the criteria is. were you in his party, max level, holy offspec, etc. 

here's the log so you can see for yourself tho: https://classic.warcraftlogs.com/reports/nLbGd2QzRHyxkVCq#type=auras&fight=16&source=3

and one for holy priest:
![image](https://github.com/user-attachments/assets/68045d11-4ad5-4f76-bc72-a7a636e6ae8e)
https://classic.warcraftlogs.com/reports/QAxGcThXM4F9z6ba#type=healing&fight=6&pins=0%24Separate%24%23244F4B%24auras-gained%240%240.0.0.Any%2465847809.0.0.Priest%24true%240.0.0.Any%24false%2447788%5E0%24Separate%24%23909049%24auras-cast%24-2%240.0.0.Any%2465847809.0.0.Priest%24true%240.0.0.Any%24false%2447753&view=events&start=2431204&end=2462933&ability=33110&source=9

**Comment by fkelava** [comment]
Yeah. In his party, max level, shadow MS, disc OS. The only difference between us should be that you were in raid, and I was not.

I'll test during our main raid later tonight and report my findings.

**Comment by Nexes00** [comment]
As far as I know this is an issue related to Prayer of Mending's eating each other.  For example the disc priest has a PoM on the target, then a holy or shadow priest overwrites that Prayer of Mending with their own.  It doesn't provide the large benefit that would appear at first glance and why you wouldn't see divine aegis on a holy priest PoM crit proc'd from SWD while standing at a target dummy.  For reproducibility you would need two priests, one of which has divine aegis, and have them overwrite the disc priests PoM.

**Comment by JamminL** [comment]
Archiving old non-relevant issues in preparation for MOP. If I made a mistake please comment on it.
